frontend/javascripts/oxalis/model/sagas/isosurface_saga.ts

```diff
@@ -546,17 +545,42 @@ function* loadPrecomputedMeshForSegmentId(
   yield* put(addPrecomputedIsosurfaceAction(layerName, id, seedPosition, meshFileName));
   yield* put(startedLoadingIsosurfaceAction(layerName, id));
   const dataset = yield* select((state) => state.dataset);
+
   let availableChunks = null;
 
+  const meshFile = yield* select((state) =>
+    (state.localSegmentationData[layerName].availableMeshFiles || []).find(
+      (file) => file.meshFileName === meshFileName,
+    ),
+  );
+  if (!meshFile) {
+    throw new Error("Could not find requested mesh file.");
+  }
+
+  const version = meshFile.formatVersion;
   try {
-    availableChunks = yield* call(
-      getMeshfileChunksForSegment,
-      dataset.dataStore.url,
-      dataset,
-      getBaseSegmentationName(segmentationLayer),
-      meshFileName,
-      id,
-    );
+    // todo: should actually check against 3, but some new mesh files
```

Comment:
todo

Reply:
> For the version comparison with 2, I'm a bit hesitant, since I know that there are already mesh files in circulation with v=2. So, maybe it's alright to leave the comparison at >= 2. What do you think?

Are we talking about many (or important) mesh files with version 2? Otherwise, I would lean towards bumping the comparison to 3 and assert that the mesh file version is neither 1 nor 2. This would make the code easier to understand and forfeit the need to explain that everything is named v3 but some mesh files have v2. Also we would need to make sure that there exists a test dataset that has a mesh file with the correct version 3.